collective/noindexing/testing.py

```diff
@@ -1,22 +1,27 @@
+from plone.testing import Layer
 from plone.app.testing import IntegrationTesting
-from plone.app.testing import PLONE_FIXTURE
-from plone.app.testing import PloneSandboxLayer
-from zope.configuration import xmlconfig
+from plone.app.testing import PloneWithPackageLayer
```

Comment:
I have not noticed `PloneWithPackageLayer` before. Looks useful.
Note to self: start using this more.

Reply:
For a reason I do not get, ``PloneWithPackageLayer`` is not used by Timo and friends. Disclaimer: @jfroche and me built ``PloneWithPackageLayer``.